src/solvers/dg/3d/dg.jl

```diff
@@ -2125,17 +2129,40 @@ function calc_mortar_flux!(surface_flux_values, dg::Dg3D, mortar_type::Val{:l2},
       end
     end
 
-    @. fstar_upper_left  += noncons_diamond_upper_left
-    @. fstar_upper_right += noncons_diamond_upper_right
-    @. fstar_lower_left  += noncons_diamond_lower_left
-    @. fstar_lower_right += noncons_diamond_lower_right
+    # the code below is equivalent to
```

Comment:
Ah, too bad, I was happy to see the performance of the dot syntax be on par with our handwritten loops ;-)